omero/developers/Web/CreateApp.txt

```diff
@@ -67,51 +69,62 @@ OR you could simply choose an existing location:
 Create and check out a new GitHub repository OR manually create a new directory
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
--  Login to your GitHub account homepage (e.g.
-   `<https://github.com/>`_\ <your-name>/) and click
+-  Login to your GitHub account homepage e.g.
+   `<https://github.com/>`_\<your-name>/) and click
    "New repository"
--  Enter the name of <your-app>, add a description and choose to add a
-   README.
-
--  Check out your new repository (into a new directory)
 
-   ::
+-  Enter the name of your repository e.g. ``<organisation-appname>``,
+   add a ``README.rst`` file (using the .rst extension allows it to be rendered correctly on PyPI).
+   Its contents should cover the goal of the app and the configuration instructions.
 
-       $ git clone git@github.com:<your-name>/<your-app>.git
+-  Check out your new repository into a new directory::
 
+       $ git clone git@github.com:<your-name>/<organisation-appname>.git
 
--  OR: If you have not used git to create your app directory above, then
 
-   ::
+-  OR: If you have not used git to create your app directory above, then::
 
-        $ mkdir <your-app>
+        $ mkdir <organisation-app>
 
--  In either case, you should now have a directory called ``your-app`` within
+-  In either case, you should now have a directory called ``organisation-appname`` within
    a directory that is on your :envvar:`PYTHONPATH`.
 
+Make your app installable from PyPI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is not required but it is highly recommended to make your app
+installable from PyPI. If you opt to do so, a few files need to be added:
+
+- ``MANIFEST.in`` - a file is needed in certain cases to package files not automatically included
+
+- ``setup.cfg`` - a configuration file that contains option defaults for ``setup.py`` commands
+
+- ``setup.py`` - a set-up file used to configure various aspects of the app and also used as a command line interface for packaging the app.
+
+
+See `Packaging and Distributing Projects <https://packaging.python.org/distributing/>`_ for more details. 
 
 Add the essential files to your app
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
--  Create an empty file ``<your-app>/__init__.py`` (NB: both double
-   underscores)
--  Create :file:`urls.py`
+-  Create a directory ``<organisation_appname>`` (note the underscore) to add the essential files to
+   e.g. ``omero_webtest``
```

Comment:
I would move the 'e.g. omero-webtest' to earlier in the sentence:
"Create a directory ``<organisation_appname>`` e.g. ``omero_webtest`` (note the underscore) to add the essential files to